common/modules/olm/manifests/amq-streams.v1.5.0.clusterserviceversion.yaml

```
@@ -309,24 +329,18 @@ spec:
     Red Hat AMQ Streams provides a way to run an [Apache Kafka](https://kafka.apache.org) cluster on [OpenShift](https://www.openshift.com/) in various deployment configurations.
     See the Red Hat AMQ [website](https://access.redhat.com/products/red-hat-amq) for more details about the project.
 
-    ### What is new in AMQ Streams 1.4
-
-    * Add support for Keycloak authorization
+    ### What is new in AMQ Streams 1.5
 
-    * Add support for Mirror Maker 2.0
+    * Suppor for Apache Kafka 2.5.0
```

```suggestion
    * Support for Apache Kafka 2.5.0
```

Sorry, I cannot stop :-o

---

If I make stupid mistakes it's not your fault :-)

---

This is actually a typo in the upstream CSV already :-(